src/index.ts

```diff
@@ -138,6 +138,20 @@ export const createInstance = (config: Config): Instance => {
 
       return tx;
     },
+    cancelBid: async (
```

Comment:
Refer to the task card for this function: https://zer0.io/a/network/tasks/board/fe25e18f-04ff-4832-b213-10d75ce18d92/4133f0ed-fd45-4d43-b086-7c1fb8a1cee4

This needs to cancel the bid both via the API and the Smart Contract.
Cancelling via smart contract should also be an optional thing to do (API is mandatory, Smart Contract is optional)